src/filters/BlendImage.ts

```diff
@@ -39,20 +39,27 @@ export const blendImageDefaultValues: Partial<TClassProperties<BlendImage>> = {
  *
  * const filter = new BlendImage({
  *  image: fabricImageObject,
- *  mode: 'multiply',
- *  alpha: 0.5
+ *  mode: 'multiply'
  * });
  * object.filters.push(filter);
  * object.applyFilters();
  * canvas.renderAll();
  */
 export class BlendImage extends BaseFilter {
   /**
-   * Color to make the blend operation with. default to a reddish color since black or white
-   * gives always strong result.
+   * Image to make the blend operation with.
```

Comment:
I would just call this FabricImage instead of Image, and i would specify that FabricImage transform properties are taken in consideration